### ISSUE TITLE
About the fine-tuning train_loss_conf_depth hits a plateau

### ISSUE THREAD

**Issue by qwq-vegetabledog** [facebookresearch/vggt#480]
Hi, thank you for this great work!

I have been fine-tuning VGGT on a small dataset of 2000 images with 2x A100 40GB GPUs. I have frozen the aggregator head, the train_loss_conf_depth now decreases from 0.75 to 0.35 but then the loss hits a plateau after 500 steps.  It always stagnates around 0.35-0.40.

Learning rate is 1e-5 (by the way I have tried all the lr it doesn't work)

Do you have any solution or idea for this phenomenon?

Thank you!
